precompile/contract/interfaces.go

```diff
@@ -73,6 +68,7 @@ type Configurator interface {
 		chainConfig precompileconfig.ChainConfig,
 		precompileconfig precompileconfig.Config,
 		state StateDB,
-		blockContext ConfigurationBlockContext,
+		blockNumber *big.Int,
+		blockTime uint64,
```

Comment:
Do we need this blockTime at the end? It doesn't seem used anywhere in subnet-evm. Waiting for a reply on https://github.com/ava-labs/subnet-evm/pull/1429/

Reply:
Let's keep them for symmetry, previously exposed interfaces, and possible future use, imo.
(same comments here https://github.com/ava-labs/subnet-evm/pull/1429#discussion_r1927232609)